docs/installation/upgrade/2.4-to-2.4.x.md

```diff
@@ -53,6 +53,7 @@ changed to `2.4.3`.
 | ------- | ----------------- | ----------------- |
 | [[setting,lazy_expunge_only_last_instance]] | `no` | `yes` |
 | [[setting,mail_access_groups]] |  | [[setting,default_internal_group]] |
+| [[setting,passdb_default_password_scheme]] | `PLAIN` | `CRYPT` |
```

Comment:
This could say in some way which passdbs are actually changed, since default wasn't PLAIN for all of them.